etna/records/models.py

```diff
@@ -118,24 +99,13 @@ def reference_number(self) -> str:
                 return item["reference_number"]
             except KeyError:
                 pass
-        raise ValueExtractionError(
-            f"'reference_number' could not be extracted from source data: {self._raw}"
-        )
+        return ""
 
-    def has_reference_number(self) -> bool:
-        """
-        Returns `True` if a 'reference_number' value can be extracted from the
-        raw data for this record. Otherwise `False`.
-        """
-        try:
-            self.reference_number
-        except ValueExtractionError:
-            return False
-        else:
-            return True
+    def reference_prefixed_title(self):
+        return f"{self.reference_number or 'N/A'} - {self.summary_title}"
```

Comment:
How about calling it  `reference_prefixed_summary_title` - due to return containing `self.summary_title`